client/components/suggestions/index.jsx

```diff
@@ -107,7 +107,10 @@ const Suggestions = React.createClass( {
 	},
 
 	onMouseDown( event ) {
-		this.props.suggest( event.target.textContent );
+		event.stopPropagation();
+		event.preventDefault();
+		//Additional empty space at the end adds fluidity to workflow
+		this.props.suggest( event.target.textContent + ' ' );
```

Comment:
Good question. The answer is no :) This extra space has nothing to do with how stuff looks. It just indicates to other part of the code that it can start to show more suggestions to the user. If the space was not added the cursor would be after the last character of suggestion, and this would mean that the suggestions would have nothing to suggest. Additional space pushes cursor to the empty space which can be filled with next token either manually or with suggestion. 
It is hard to explain it in writing. If you would like me to clarify or explain more I will be more than happy to do so.